modules/monitoring-contents-of-the-alerting-ui.adoc

```diff
@@ -7,43 +7,43 @@
 
 This section shows and explains the contents of the Alerting UI, a web interface to the Alertmanager.
 
-The main three pages of the Alerting UI are the *Alerts*, the *Silences*, and the *YAML* pages.
+The Alerting UI is accessible by clicking *Monitoring* -> *Alerting* in the {product-title} web console.
 
-The *Alerts* page is accessible by clicking *Monitoring* -> *Alerting* -> *Alerts* in the {product-title} web console.
+The main three pages of the Alerting UI are the *Alerts*, the *Silences*, and the *Alerting rules* pages.
+
+The *Alerts* page:
 
 image::monitoring-alerts-screen.png[]
 
 . Filtering alerts by their names.
 . Filtering the alerts by their states. To fire, some alerts need a certain condition to be true for the duration of a timeout. If a condition of an alert is currently true, but the timeout has not been reached, such an alert is in the *Pending* state.
 . Alert name.
 . Description of an alert.
-. Current state of the alert and when the alert went into this state.
 . Value of the Severity label of the alert.
+. Current state of the alert and when the alert went into this state.
 . Actions you can do with the alert.
 
-The *Silences* page is accessible by clicking *Monitoring* -> *Alerting* -> *Silences* in the {product-title} web console.
+The *Silences* page:
 
 image::monitoring-silences-screen.png[]
 
 . Creating a silence for an alert.
 . Filtering silences by their name.
 . Filtering silences by their states. If a silence is pending, it is currently not active because it is scheduled to start at a later time. If a silence expired, it is no longer active because it has reached its end time.
 . Description of a silence. It includes the specification of alerts that it matches.
-. Current state of the silence. For active silences, it shows when it ends, and for pending silences, it shows when it starts.
 . Number of alerts that are being silenced by the silence.
+. Current state of the silence. For active silences, it shows when it ends, and for pending silences, it shows when it starts.
+. User that created the silence.
 . Actions you can do with a silence.
 
-The *YAML* page is accessible by clicking *Monitoring* -> *Alerting* -> *YAML* in the {product-title} web console.
+The *Alerting Rules* page:
 
-image::monitoring-yaml-screen.png[]
+image::monitoring-alerting-rules-screen.png[]
 
-. Upload a file with Alertmanager configuration.
-. Examine and edit the current Alertmanager configuration.
-. Save the updated Alertmanager configuration.
+. Filtering Alerting Rules by their name.
+. Filtering Alerting Rules by their states.
+. Alerting Rule name.
+. Value of the Severity label of the Alerting Rule.
+. State of the Alert defined by the Alerting Rule.
 
 Also, next to the title of each of these pages is a link to the old Alertmanager interface.
```

Comment:
I don't think we should say `old` here. This is a link to Alertmanager's own UI. Maybe just `link to the Alertmanager interface`?